projects/azure/base-images/test-base.yaml

```diff
@@ -48,6 +48,8 @@ items:
         # TODO: I should not be here. Remove me.
         RUN go get github.com/go-bindata/go-bindata && \
             go get golang.org/x/tools/cmd/goyacc && \
+            go get golang.org/x/tools/cmd/goimports && \
+            go get golang.org/x/lint/golint && \
```

Comment:
We don't need golint in here; if you want to lint PRs you can always comment `/lint` on PRs